docs/built_in_policy.md

```diff
@@ -0,0 +1,32 @@
+# Policy Properties
+
+There are built-in property names that can be used in the policies.
+anax will fill in the properties during the agreement negotiation process.
+For the node properties, the agent will introspect the node for the values of those properties.
+The user defined policies (deployment policy, model policy, service policy) need to add constraints on these properties if needed.
+
+## Built-in properties
+
+* for node policy
+
+**Name** | **Description** | **Possible values**
+----- | ----- | -----
+openhorizon.cpu | The number of CPUs (will be fetched from /proc/cpuinfo file) | `int` e.g. 4
+openhorizon.memory| The amount of memory in MBs (will be fetched from /proc/meminfo)| `int` e.g. 1024
+openhorizon.arch| The hardware architecture of the node (will be fetched from GOARCH)| `string` e.g. amd64
+openhorizon.hardwareId| The device serial number if it can be found (will be fetched from /proc/cpuinfo). A generated Id otherwise. | `string`
+openhorizon.allowPrivileged| Property set to determine if privileged services may be run on this device. Can be set by user, default is false. This is the only writable node property| `boolean` 
+openhorizon.kubernetesVersion| Kubernetes version of the cluster the agent is running in| `string` e.g. 1.18
+
+**Note:Provided properties (except for allowPrivileged) are read-only, the system will ignore updating of the node policy and changing any of the built-in properties*    
+
+* for service policy
+
+**Name** | **Description** | **Possible values**
+----- | ----- | -----
+openhorizon.service.url | The unique name of the service (comes from `url` field of service definition) | `string` e.g. https://someOrg/someService
+openhorizon.service.name| The unique name of the service (comes from `url` field of service definition)| `string` e.g. MyService
+openhorizon.service.org| The multi-tenant org where the service is defined. The org defaults to the org of the node (comes from `org` field of service definition) | `string` e.g. MyOrg
```

Comment:
Where did you get the info about this field defaulting? I dont think it does. It is always the org that the service is defined in.

Reply:
Do we need that sentence to remain in this doc?

Reply:
I think the way you have it worded right now is fine.